mimic/test/test_resource.py

```diff
@@ -284,4 +283,4 @@ def test_verbose_logging(self):
         self.assertNotEqual(None, response_match)
         self.assertEqual(url, response_match.group('url'))
         headers = json.loads(response_match.group('headers'))
-        self.assertEqual(['application/json'], headers.get('Content-Type'))
+        self.assertEqual(['close'], headers.get('Connection'))
```

Comment:
NICE catch, @ksheedlo !  I typo'd `requestHeaders` in the implementation where it should have said `responseHeaders` and hence we were logging the wrong headers.  Really glad you were able to review this! :smile: